### PR TITLE
Update add-domain.md

### PR DESCRIPTION
**Pull request by johnmeilleur**
Update documentation to reflect that a least privileged role to add a domain exist.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/setup/add-domain.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ description: "Use the setup wizard to add your domain to Microsoft 365 in the Mi
   
 ## Before you begin
 
-To Add, modify or remove domains you **must** be a **Global Administrator** of a [business or enterprise plan](https://products.office.com/business/office). These changes affect the whole tenant; *Customized administrators* or *regular users* won't be able to make these changes.
+To add, modify or remove domains you **must** be a **Domain Name Administrator** or **Global Administrator** of a [business or enterprise plan](https://products.office.com/business/office). These changes affect the whole tenant; *Customized administrators* or *regular users* won't be able to make these changes.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/add-domain.md]:
```suggestion
To add, modify, or remove domains, you **must** be a **Domain Name Administrator** or **Global Administrator** of a [business or enterprise plan](https://products.office.com/business/office). These changes affect the whole tenant; *Customized administrators* or *regular users* won't be able to make these changes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @johnmeilleur, kindly commit the minor suggested change. Many thanks.

**Comment by johnmeilleur** [comment]
@denisebmsft @yogkumgit I believe this change is ready to go.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks

**Pull request merged**